charmcraft/commands/build.py

```diff
@@ -178,12 +178,29 @@ def run(self) -> List[str]:
         """
         charms: List[str] = []
 
+        # Validate base indices, if any.
+        if bases_indices:
```

Comment:
If possible, I'd try to validate command line parameters as soon as possible. In this case, where it can not really be done with argparse (we could validate it's >=0, but at parsing time we don't know the bases length), I'd validate this in the first lines of the Command's `run` method.

Reply:
Done!